pandas/core/groupby/ops.py

```diff
@@ -259,6 +266,9 @@ def get_out_dtype(self, dtype: np.dtype) -> np.dtype:
                 out_dtype = "object"
         return np.dtype(out_dtype)
 
+    def uses_mask(self) -> bool:
```

Comment:
this is very far from actual use, would remove

Reply:
Think this falls under the umbrella of your comment here https://github.com/pandas-dev/pandas/pull/40651#discussion_r612400301. I liked folding this functionality into `WrappedCythonOp` since it only depends on the specific op. And `_MASKED_CYTHON_FUNCTIONS` can then just act like a constant to hold which algos have masked support

Reply:
I agree having this in `WrappedCythonOp` class makes sense. This class also holds other information on the cython algos like `_CYTHON_FUNCTIONS` with a mapping of ops name to cython function names.